### PR TITLE
feat(cmd): .simbridgedebug command

### PR DESCRIPTION
**Pull request by Lauvaan**
## Description

Fixes: #473 

A command that explains the reason for simbridge crashing, and provides steps to send a screenshot that support can use to debug the situation.

## Test Results

(don't have an image server up(and didn't bother setting up one rn) to showcase the image, but it should work)
image

### CODE REVIEW

**Comment by Lauvaan** [comment]
Going to get screenshots and add changelog after I get back home, now have only chromebook and github.dev, so can't take the screenshots.

**Comment by pdellaert** [comment]
@Lauvaan, can you add a CHANGELOG entry? and have a quick look at the lint error? (seems like there's unnecessary escapes)

**Comment by Lauvaan** [comment]
> @Lauvaan, can you add a CHANGELOG entry? and have a quick look at the lint error? (seems like there's unnecessary escapes)

Yep, gonna do it on the weekend probably, if I have time

File: src/commands/support/simbridgeDebug.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const SIMBRIDGE_HELP_URL = `${imageBaseUrl}/support/simbridgeDebug.png`;
+
+const simbridgeDebugEmbed = makeEmbed({
+    title: 'FlyByWire Support | Why does SimBridge crash?',
+    description: makeLines([
+        'If SimBridge crashes moments after start, it could be due to bad installation of SimBridge, missing modules or another issue. In order to assist you further we will need to see your SimBridge logs.',
+        '',
+        '1. Open the folder where SimBridge is installed.',
+        '2. Open a terminal window in that location (Hold `SHIFT` and right click on the empty space of the folder, then select "Open Windows Terminal", "Open Command Prompt", or "Open Powershell Window".',
+        '3. Type `fbw-simbridge.exe` and press enter',
```
Comment (NUTTTTTTTT) [inline src/commands/support/simbridgeDebug.ts:14]:
```suggestion
        '3. Type `fbw-simbridge.exe` and press enter.',
```

**Review by NUTTTTTTTT — CHANGES_REQUESTED** [review]
Aside from this, LGTM x

**Comment by benw202** [comment]
> Aside from this, LGTM x

Thanks! Can you merge when approved pls

**Pull request merged**